search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -117,23 +139,43 @@ function setTimeRange(timeRange) {
     return;
   }
   currentTimeRange = timeRange;
-  updateTrends();
+  getIsWorldLevel() ?
+      updateGlobalTrendsAndDisplayFirst() : updateUsTrendsAndDisplayFirst();
 }
 
 /**
- * Fetches current top trends from back end and displays them on the website.
+ * Fetches current top global trends from the backend, displays them on the 
+ * website, and shows map data for the first trend.
  * @param {boolean=} setNewTrendEnabled Boolean for whether or not to change
  *     the trend that the user is viewing.
  */
-function updateTrends(setNewTrendEnabled = true) {
+function updateGlobalTrendsAndDisplayFirst(setNewTrendEnabled = true) {
```

Comment:
```suggestion
function updateGlobalTrendsAndInitiallyDisplayFirstTrend(setNewTrendEnabled = true) {
```
altho long, like Haylee mentioned before method names can be as long as you want

Reply:
resolved